src/Containers/OvalTable/OvalTable.stories.tsx

```diff
@@ -0,0 +1,318 @@
+import {Meta, Story} from "@storybook/react";
+import {createStoryTitle} from "../../Constants";
+import {IOvalTable, OvalTable} from "@Containers/OvalTable/OvalTable";
+import React from "react";
+
+export default {
+    title: createStoryTitle('OvalTable'),
+    component: OvalTable,
+} as Meta;
+
+const Template: Story<IOvalTable> = (args) => <OvalTable {...args} />;
+
+/**
+ * Prints the Selected Child index to the console when Chair is clicked
+ * @param tableIndex
+ * @param chairIndex
+ * @param selectedTableIndex
+ */
+const handleOnChairClick = (
```

Comment:
Remove this and replace the console log with storybook actions so it emits the correct event.

action("Log Data")